_meta/beat.yml

```
@@ -247,6 +239,82 @@ apm-server:
     # Configure curve types for ECDHE based cipher suites.
     #ssl.curve_types: []
 
+  #---------------------------- APM Server - ILM Index Lifecycle Management ----------------------------
+
+  # Index Lifecycle Mangement (ILM)
+  #ilm:
+    # `ilm.enabled: "auto"`:
```

```suggestion
    # `enabled: "auto"`
```
same for all others bellow: remove `ilm` duplicated and extra `:` at the end